src/reducers/teamsReducer.js

```diff
@@ -0,0 +1,17 @@
+export default (
+  state = {
+    urlName: 'raptors', // TODO: set team based on user's geographical location
```

Comment:
move this `urlName` inside of `selectedTeam` object. 
```
state = {
  teams: [],
  selectedTeam: {
    urlName: 'raptors', // TODO: set team based on user's geographical location
  },
}
```

Otherwise, if the user goes to `localhost:3000`, instead it will redirect to `http://localhost:3000/undefined/players`